packages/repack/src/modules/ScriptManager/__tests__/ScriptManager.test.ts

```diff
@@ -435,6 +437,7 @@ describe('ScriptManagerAPI', () => {
       'src_App_js',
       'main'
     );
+    await ScriptManager.shared.loadScript('src_App_js', 'main');
```

Comment:
why do we need to call `loadScript` here in test for `resolveScript` ?

Reply:
It is because we move `saveCache` from `resolveScriopt` to end of `loadScript`. So if we want to sure caching system working correctly, we should call `loadScript` to save cache

Reply:
do we need to call `resolveScript` before here? I think we can just do `loadScript` now

Reply:
We need value returned by `resolveScript` to `script1` to check fetch

Reply:
ah right, makes sense!